pages/aws/config/custom_overrides.mdx

```diff
@@ -129,16 +129,16 @@ export function middleware(request: NextRequest) {
 
   // ... your code here
 }
- 
+
 ```
 
 
 ### Use middy.js with the wrapper
 
 ```ts
 // customWrapper.ts
-import streamingWrapper from 'open-next/wrappers/aws-lambda.js'
-import {WrapperHandler} from 'open-next/types/open-next'
+import streamingWrapper from '@opennextjs/aws/overrides/wrappers/aws-lambda.js'
+import type { WrapperHandler } from '@opennextjs/aws/types/overrides'
```

Comment:
We should either always add the `.js` at the end or never. 
It's probably better to add it because a lot of config will complain without it, but without explaining exactly what's the error